drivers/eks/templates.go

```diff
@@ -464,13 +496,14 @@ Resources:
         PropagateAtLaunch: 'true'
     UpdatePolicy:
       AutoScalingRollingUpdate:
-        MinInstancesInService: '1'
         MaxBatchSize: '1'
+        MinInstancesInService: !Ref NodeAutoScalingGroupDesiredCapacity
+        PauseTime: 'PT5M'
 
   NodeLaunchConfig:
     Type: AWS::AutoScaling::LaunchConfiguration
     Properties:
-      AssociatePublicIpAddress: !Ref PublicIp
+      AssociatePublicIpAddress: 'true'
```

Comment:
Won't hardcoding this to `true` make the `AssociateWorkerNodePublicIP` option useless?